packages/mdc-chips/chip/index.js

```diff
@@ -125,6 +125,9 @@ class MDCChip extends MDCComponent {
       notifyInteraction: () => this.emit(strings.INTERACTION_EVENT, {chip: this}, true /* shouldBubble */),
       notifyTrailingIconInteraction: () => this.emit(
         strings.TRAILING_ICON_INTERACTION_EVENT, {chip: this}, true /* shouldBubble */),
+      getComputedStyleValue: (propertyName) => window.getComputedStyle(this.root_).getPropertyValue(propertyName),
+      setStyleProperty: (propertyName, value) => this.root_.style.setProperty(propertyName, value),
+      removeFromDOM: () => this.root_.remove(),
```

Comment:
This won't work in IE. You're going to have to do `this.root_.parentNode.removeChild(this.root_)`